ui/apos/components/AposExportModal.vue

```diff
@@ -229,7 +229,7 @@ export default {
       this.relatedTypes = await apos.http.get('/api/v1/@apostrophecms/import-export/related', {
         busy: true,
         qs: {
-          type: this.type
+          types: [ this.type ]
```

Comment:
New related route takes an array of types (for future pages implementation).

Reply:
could be good to test the route in addition of the method, but lgtm